<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>711</width>
    <height>416</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QPushButton" name="Button_A">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>100</y>
      <width>41</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>A</string>
    </property>
   </widget>
   <widget class="QPushButton" name="Button_aa">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>100</y>
      <width>41</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Ą</string>
    </property>
   </widget>
   <widget class="QPushButton" name="Button_B">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>100</y>
      <width>41</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>B</string>
    </property>
   </widget>
   <widget class="QPushButton" name="Button_cc">
    <property name="geometry">
     <rect>
      <x>260</x>
      <y>100</y>
      <width>41</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Ć</string>
    </property>
   </widget>
   <widget class="QPushButton" name="Button_D">
    <property name="geometry">
     <rect>
      <x>310</x>
      <y>100</y>
      <width>41</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>D</string>
    </property>
   </widget>
   <widget class="QPushButton" name="Button_C">
    <property name="geometry">
     <rect>
      <x>210</x>
      <y>100</y>
      <width>41</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>C</string>
    </property>
   </widget>
   <widget class="QPushButton" name="Button_ee">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>140</y>
      <width>41</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Ę</string>
    </property>
   </widget>
   <widget class="QPushButton" name="Button_F">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>140</y>
      <width>41</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>F</string>
    </property>
   </widget>
   <widget class="QPushButton" name="Button_I">
    <property name="geometry">
     <rect>
      <x>310</x>
      <y>140</y>
      <width>41</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>I</string>
    </property>
   </widget>
   <widget class="QPushButton" name="Button_G">
    <property name="geometry">
     <rect>
      <x>210</x>
      <y>140</y>
      <width>41</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>G</string>
    </property>
   </widget>
   <widget class="QPushButton" name="Button_E">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>140</y>
      <width>41</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>E</string>
    </property>
   </widget>
   <widget class="QPushButton" name="Button_H">
    <property name="geometry">
     <rect>
      <x>260</x>
      <y>140</y>
      <width>41</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>H</string>
    </property>
   </widget>
   <widget class="QPushButton" name="Button_K">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>180</y>
      <width>41</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>K</string>
    </property>
   </widget>
   <widget class="QPushButton" name="Button_L">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>180</y>
      <width>41</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>L</string>
    </property>
   </widget>
   <widget class="QPushButton" name="Button_N">
    <property name="geometry">
     <rect>
      <x>310</x>
      <y>180</y>
      <width>41</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>N</string>
    </property>
   </widget>
   <widget class="QPushButton" name="Button_ly">
    <property name="geometry">
     <rect>
      <x>210</x>
      <y>180</y>
      <width>41</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Ł</string>
    </property>
   </widget>
   <widget class="QPushButton" name="Button_J">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>180</y>
      <width>41</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>J</string>
    </property>
   </widget>
   <widget class="QPushButton" name="Button_M">
    <property name="geometry">
     <rect>
      <x>260</x>
      <y>180</y>
      <width>41</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>M</string>
    </property>
   </widget>
   <widget class="QPushButton" name="Button_O">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>220</y>
      <width>41</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>O</string>
    </property>
   </widget>
   <widget class="QPushButton" name="Button_u_ZAM">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>220</y>
      <width>41</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Ó</string>
    </property>
   </widget>
   <widget class="QPushButton" name="Button_S">
    <property name="geometry">
     <rect>
      <x>310</x>
      <y>220</y>
      <width>41</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>S</string>
    </property>
   </widget>
   <widget class="QPushButton" name="Button_P">
    <property name="geometry">
     <rect>
      <x>210</x>
      <y>220</y>
      <width>41</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>P</string>
    </property>
   </widget>
   <widget class="QPushButton" name="Button_ni">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>220</y>
      <width>41</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Ń</string>
    </property>
   </widget>
   <widget class="QPushButton" name="Button_R">
    <property name="geometry">
     <rect>
      <x>260</x>
      <y>220</y>
      <width>41</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>R</string>
    </property>
   </widget>
   <widget class="QPushButton" name="Button_T">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>260</y>
      <width>41</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>T</string>
    </property>
   </widget>
   <widget class="QPushButton" name="Button_U">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>260</y>
      <width>41</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>U</string>
    </property>
   </widget>
   <widget class="QPushButton" name="Button_Z">
    <property name="geometry">
     <rect>
      <x>310</x>
      <y>260</y>
      <width>41</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Z</string>
    </property>
   </widget>
   <widget class="QPushButton" name="Button_W">
    <property name="geometry">
     <rect>
      <x>210</x>
      <y>260</y>
      <width>41</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>W</string>
    </property>
   </widget>
   <widget class="QPushButton" name="Button_si">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>260</y>
      <width>41</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Ś</string>
    </property>
   </widget>
   <widget class="QPushButton" name="Button_Y">
    <property name="geometry">
     <rect>
      <x>260</x>
      <y>260</y>
      <width>41</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Y</string>
    </property>
   </widget>
   <widget class="QPushButton" name="Button_rzy">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>300</y>
      <width>41</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Ż</string>
    </property>
   </widget>
   <widget class="QPushButton" name="Button_zi">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>300</y>
      <width>41</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Ź</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_haslo">
    <property name="geometry">
     <rect>
      <x>370</x>
      <y>300</y>
      <width>301</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>20</pointsize>
     </font>
    </property>
    <property name="layoutDirection">
     <enum>Qt::LeftToRight</enum>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="label_kategoria">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>20</y>
      <width>91</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Kategoria:</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_wylosowane_kat">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>20</y>
      <width>191</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="Button_Graj">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>300</y>
      <width>141</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Zagraj</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_grafika">
    <property name="geometry">
     <rect>
      <x>380</x>
      <y>30</y>
      <width>231</width>
      <height>261</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label_bledy">
    <property name="geometry">
     <rect>
      <x>310</x>
      <y>300</y>
      <width>47</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>711</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
